R/modules.R

```diff
@@ -405,44 +408,40 @@ module_labels <- function(modules) {
 #'
 #' @param x (`teal_modules`) to print
 #' @param indent (`integer`) indent level;
-#'   each `submodule` is indented one level more
-#' @param ... (optional) additional parameters to pass to recursive calls of `toString`
+#'   each nested `teal_modules` or `teal_module` is indented one level more
 #' @return (`character`)
 #' @export
 #' @rdname modules
-toString.teal_modules <- function(x, indent = 0, ...) { # nolint
+format.teal_modules <- function(x, indent = 0, ...) { # nolint
   # argument must be `x` to be consistent with base method
   paste(c(
     paste0(rep(" ", indent), "+ ", x$label),
-    unlist(lapply(x$children, toString, indent = indent + 1, ...))
+    unlist(lapply(x$children, format, indent = indent + 1, ...))
   ), collapse = "\n")
 }
 
 #' Converts `teal_module` to a string
 #'
-#' @inheritParams toString.teal_modules
+#' @inheritParams format.teal_modules
 #' @param x `teal_module`
-#' @param ... ignored
 #' @export
 #' @rdname module
-toString.teal_module <- function(x, indent = 0, ...) { # nolint
+format.teal_module <- function(x, indent = 0, ...) { # nolint
   paste0(paste(rep(" ", indent), collapse = ""), "+ ", x$label, collapse = "")
 }
 
 #' Prints `teal_modules`
 #' @param x `teal_modules`
-#' @param ... parameters passed to `toString`
 #' @export
 #' @rdname modules
 print.teal_modules <- function(x, ...) {
-  s <- toString(x, ...)
-  cat(s)
-  return(invisible(s))
+  cat(format(x, ...))
+  invisible(x)
 }
 
 #' Prints `teal_module`
 #' @param x `teal_module`
-#' @param ... parameters passed to `toString`
+#' @param ... arguments passed to `format`
```

Comment:
This leaves `...` in `format` undocumented.

Reply:
aaah right. changed in here so they have generic message b01b57207c2b538e56ecbf8dfca2cac3d3e11fa6
`arguments passed to other methods.`